### DISCUSSION TITLE
How to use Peavy 24 channel board with Jamulus

### DISCUSSION THREAD

**Discussion by Niceguy12342006** [jamulussoftware/jamulus#2568] [General]
Last year, our bass player was diagnosed with a serious illness. He was not able to be in rehearsals for months. As he got better, he was feeling good enough to play, but was not released from home. 
     I was reading about free software called Jamulus (see llcon.sourceforge.net). It is software that was developed during covid, for bands to have all band members rehearse from home. We were a bit different in that 4 members were able to rehearse in our studio.
As I’m looking at you tube videos and following directions, it appeared the version had been updated, to make the installation easier. 
You do need to download a ASIO driver, if your computer doesn’t have. Our ill band member figured out how to establish a server. 
Setting up a server in Jamulus
    1.	Open the Jamulus from your programs
    2.	Click on Jamulus server
    3.	From this dialog box fill in the server setup tabs
    4.	Click the options tab and check the box stat minimized on Windows startup.
NOTE: when you shut down your computer your server will not be displayed in Jamulus on restart unless after you restart your computer you open up the Jamulus server again and repeat step 4. Then shut down your computer and restart it again.
I don’t know why the server your setup doesn’t stay when you shut down after being on Jamulus, but it does. To speed up this part I placed a shortcut to the Jamulus server on my desk top. 
     He used a small analog mixer. He was able to plug his bass guitar(must have a battery) into one channel, and his vocal mic in the second channel. He used a rca plug out to the computer mic in. He plugged his headphones into his computer headphone jack.
     For the studio, we don’t have everyone on headphones. Normally, we run our vocals thru our Peavy24 channel analog mixer, into an amplifier, and then into our two floor monitors, using the main outs. Now for our bass player to hear the keys, guitars, and drums, I placed a mic in front of the guitar amplifiers along with one overhead drum mic. Keys we run thru the board anyway. Those mics along with vocal mics I ran a line from Aux 1 to a second amplifier, and then a line from the amplifier to an audio interface (presonus firestudio project) and then to my computer. The amplifier was set on 1, as the signal needs a bit of amplification. 
     On the Jamulus software, my volume sliders were set all way up. On my slider, I hit the mute button at the bottom (so we will hear our band member, but not to hear ourselves thru the computer) to avoid feedback. All effects were turned off on the board.
     Volume control is multifold. You have the gain on each channel, the channel Aux 1 volume control, the main Aux1 volume control, the amplifier volume control, the volume control on the interface, and the volume control on the computer software. It’s a bit of trial and error to get it right. We started with volume control at 50%, and went from there, other than the power amp (we put that on 1, and left it at 1).

**Comment by pcar75** [comment]
Just a small correction here.  Jamulus wasn't developped (i.e. started) during the pandemic . It dates back to January 2006  ( ref. https://github.com/jamulussoftware/jamulus/blob/master/ChangeLog ).  Its popularity just went exponential since the pandemic, which is great but also imposed a rapid and greater burden on it's maintainers.

There is a wealth of information in this forum and also in SourceForge.net - its previous hosting faciliy ( ref. https://sourceforge.net/p/llcon/discussion/ ).

I would add that a 'cloud' server should offer lower ping/overall delays because of possible better performance but mostly because one usually has better network and trunks available than a 'home' server.  Comments abound here (some in documentation) as to how to set up a cloud server for low costs through web service providers (AmazonWS, Google Web, Microsoft Azure, etc.).  Then the server can be private or public. 

Enjoy.
